cmd/axelard/main.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/axelarnetwork/axelar-core/cmd/axelard/cmd/utils"
 )
 
-//go:generate ./genDocs.sh ../../docs/cli
+//go:generate ./genDocs.sh ../../Makefile ../../docs/cli
```

Comment:
```suggestion
//go:generate make -f ../../Makefile docs
```
move the docs generation logic into the Makefile, so ldflags don't need to be printed out separately